### DISCUSSION TITLE
Make the translation strings translation field full width

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17134] [Feature Requests]
When creating translation strings, we have two fields: one to select the language and one where we put the translation in. Both fields are displayed `half width`. It would be nice to have them full width. Rather more it would be useful to have multi-line text, because translations can get quite long.

**Comment by azrikahar** [comment]
> Rather more it would be useful to have multi-line text, because translations can get quite long.

#13590 should help implement this particular use case 👍

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
